awesomemotive/file-upload-types
# Reduce number of phpcs errors

Fixes #45 
Fixed all errors (some false positives muted - like invalid place to register hooks etc).

## Testing procedure
- run `composer install`
- run `vendor/bin/phpcbf`
- you should not have any errors displayed
